File: src/main/java/com/lawencon/elearn/dto/quizcollection/QuizCollectionInsertReqDto.java
package com.lawencon.elearn.dto.quizcollection;

public class QuizCollectionInsertReqDto {
	private Long classDtlId;
	private Long quizScheduleId;
	private String fileCode;
	private String fileExt;
	public Long getClassDtlId() {
		return classDtlId;
	}
	public void setClassDtlId(Long classDtlId) {
		this.classDtlId = classDtlId;
	}
	public Long getQuizScheduleId() {
		return quizScheduleId;
	}
	public void setQuizScheduleId(Long quizScheduleId) {
		this.quizScheduleId = quizScheduleId;
	}
	public String getFileCode() {
		return fileCode;
	}
	public void setFileCode(String fileCode) {
		this.fileCode = fileCode;
	}
	public String getFileExt() {
		return fileExt;
	}
	public void setFileExt(String fileExt) {
		this.fileExt = fileExt;
	}
	
}

File: frontend/src/app/dto/quiz-collection/quiz-collection.insert.req.dto.ts
export class QuizCollectionInsertReqDto {
    private classDtlId! : number
	private quizScheduleId! : number
	private fileCode! : string
	private fileExt! : string

    public get _classDtlId(): number {
        return this.classDtlId;
    }

    public set _classDtlId(_classDtlId: number) {
        this.classDtlId = _classDtlId;
    }

    public get _quizScheduleId(): number {
        return this.quizScheduleId;
    }

    public set _quizScheduleId(_quizScheduleId: number) {
        this.quizScheduleId = _quizScheduleId;
    }

    public get _fileCode(): string {
        return this.fileCode;
    }

    public set _fileCode(_fileCode: string) {
        this.fileCode = _fileCode;
    }

    public get _fileExt(): string {
        return this.fileExt;
    }

    public set _fileExt(_fileExt: string) {
        this.fileExt = _fileExt;
    }

}